test/gist.spec.js

```diff
@@ -64,6 +64,47 @@ describe('Gist', function() {
             }));
          }));
       });
+
+   });
+
+   describe('comments', function() {
```

Comment:
Why don't you put this suite in the `creating/modifying` suite above? That way when we do more stuff (like deleting) a gist, it will get cleaned up too.